### ISSUE TITLE
HAS_PROPERTIES not updated automatically

### ISSUE THREAD

**Issue by rfc2822** [dmfs/opentasks#878]
Hi,

The task column `HAS_ALARMS` is automatically maintained by SQL triggers.

However, `HAS_PROPERTIES` is not set anywhere, so it's always `null`, regardless of there are properties or not.

Should `HAS_PROPERTIES` be updated by triggers too?

**Comment by dmfs** [comment]
Yes it should be updated automatically by a trigger. Properties (and alarms) are not officially released yet (mostly because there are still bits and pieces missing). I don't expect the general concept to change though.
